rocketmq-docker/4.3.0/kubernetes/deployment.yaml

```diff
@@ -11,25 +11,27 @@ spec:
     spec:
       containers:
       - name: broker
-        image: apache/rocketmq-broker:4.3.0-k8s
+        image: apache/rocketmq:4.3.0
+        command: ["sh","mqbroker", "-n","localhost:9876"]
         imagePullPolicy: IfNotPresent
         ports:
           - containerPort: 10909
           - containerPort: 10911
         volumeMounts:
-          - mountPath: /opt/logs
+          - mountPath: /root/logs
             name: brokeroptlogs
-          - mountPath: /opt/store
+          - mountPath: /root/store
             name: brokeroptstore
       - name: namesrv
-        image: apache/rocketmq-namesrv:4.3.0-k8s
+        image: apache/rocketmq:4.3.0
+        command: ["sh","mqnamesrv"]
         imagePullPolicy: IfNotPresent
         ports:
           - containerPort: 9876
         volumeMounts:
-          - mountPath: /opt/logs
+          - mountPath: /root/logs
```

Comment:
why change to root dir?

Reply:
It comes from https://github.com/apache/rocketmq/blob/master/distribution/conf/logback_broker.xml#L40

Reply:
> It comes from https://github.com/apache/rocketmq/blob/master/distribution/conf/logback_broker.xml#L40

The default dir is $HOME/logs and since your USER is root, your $HOME is /root, right?

Reply:
@harrycodawang Yes.